docs/ref/models/querysets.txt

```diff
@@ -1844,34 +1844,42 @@ they query the database each time they're called.
 .. method:: get(**kwargs)
 
 Returns the object matching the given lookup parameters, which should be in
-the format described in `Field lookups`_.
+the format described in `Field lookups`_. Normally you should use lookups
+that are guaranteed unique by the database, such as the primary key or fields
+in a unique constraint. For example::
 
-``get()`` raises :exc:`~django.core.exceptions.MultipleObjectsReturned` if more
-than one object was found. The
-:exc:`~django.core.exceptions.MultipleObjectsReturned` exception is an
-attribute of the model class.
+    Entry.objects.get(id=1)
```

Comment:
Use `pk` instead of `id`? Do we want an example with a unique constraint too?

Reply:
The other docs on this page use `id` so I'll stick with that. I have added one using a unique constraint based on an imaginary `(blog, entry_number)` constraint.